src/pages/generate.tsx

```diff
@@ -116,98 +134,362 @@ export default function ClientIdentifierGenerator() {
-                  </Grid>
-                  <Grid container item>
-                    <FormLabel>Redirect URIs</FormLabel>
-                  </Grid>
-                  <Grid item container marginLeft={3} paddingRight={3}>
-                    <FieldArray
-                      name="redirectUris"
-                      component={RedirectUrisComponent}
-                    />
-                  </Grid>
-                  <Grid item>
-                    <FormControlLabel
-                      label="Support usage of refresh tokens / offline access"
-                      control={
-                        <Checkbox
-                          name="useRefreshTokens"
-                          value={form.values.useRefreshTokens}
-                          checked={form.values.useRefreshTokens}
-                          onChange={form.handleChange}
-                        />
-                      }
-                    />
-                  </Grid>
-                  <Grid item container justifyContent="flex-end" spacing={2}>
-                    <Grid item>
-                      <Button color="secondary" variant="outlined" type="reset">
-                        Reset
-                      </Button>
+    <Grid
+      container
+      item
+      maxWidth="50em"
+      direction="column"
+      marginLeft="auto"
+      marginRight="auto"
+    >
+      <Grid container item>
+        <Grid container item justifyContent="center">
+          <Grid item>
+            <Typography variant="h4">
+              Generate a Client Identifier Document
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid container item padding={2} spacing={1} marginLeft={2}>
+          <Grid item>
+            <Typography variant="body1">
+              This generator helps you set up a Client Identifier Document. A
+              Client Identifier Document describes your application, the client,
+              to a Solid OpenID Connect Provider (Solid OIDC Provider).
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography variant="body1">
+              Do you want your users to login to their solid pods? Then you need
+              a Client Identifier Document! The user&apos;s OIDC Authentication
+              Provider will check your Client Identifier Document, let the user
+              authenticate, and hand your application an OIDC ID token.
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography variant="body1">
+              With the token, your application can request resources from the
+              user&apos;s pod.
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid
+          container
+          paddingLeft={2}
+          paddingRight={2}
+          direction="column"
+          spacing={1}
+        >
+          <Grid item>
+            <Typography variant="h5">
+              Your app&apos;s (client&apos;s) information
```

Comment:
```suggestion
              Your app&apos;s (client&apos;s) essential information
```
Either that, or minimal